java/connector-framework-internal/src/main/java/org/identityconnectors/framework/impl/serializer/CommonObjectHandlers.java

```diff
@@ -92,8 +73,7 @@ protected AttributeHandler(final Class<T> clazz, final String typeName) {
         @Override
         public final Object deserialize(final ObjectDecoder decoder) {
             final String name = decoder.readStringField("name", null);
-            @SuppressWarnings("unchecked")
-            final List<Object> value = (List) decoder.readObjectField("Values", List.class, null);
+            @SuppressWarnings("unchecked") final List<Object> value = (List) decoder.readObjectField("Values", List.class, null);

```

Comment:
Please avoid such formatting-only changes.

Reply:
Thank you, this seems again some formatting config issue of my IDE I guess, Ill fix the occurrences in the next commit as well.